packages/commonwealth/client/scripts/views/pages/Communities/helpers.ts

```diff
@@ -5,5 +5,5 @@ export const getCommunityCountsString = (totalCommunities: number) => {
     totalCommunities >= 1000
       ? numeral(totalCommunities).format('0.0a')
       : totalCommunities
-  } communities`;
+  } ${totalCommunities === 1 ? 'community' : 'communities'}`;
```

Comment:
we can use [pluralize()](https://github.com/hicommonwealth/commonwealth/blob/6884d90bb9705205514d23e56649054a2f5acec0/packages/commonwealth/client/scripts/helpers/index.tsx#L79)

Reply:
@mzparacha @masvelio done. Thanks for catching that.